src/containers/CalculatorContainer.js

```diff
@@ -1,14 +1,20 @@
 import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
 
 import Calculator from '../components/Calculator/Calculator'
-import subjectPicker from '../store/selectors/subjectPicker'
-import { PRIMARY_SUBJECTS, LINGUAL_SUBJECTS } from '../constants/SubjectTypes'
+import subjectPicker from '../store/selectors/subjectsPicker'
```

Comment:
@Fafruch You have changed file name to `subjectsPicker` but imported function is `subjectPicker `